src/main/kotlin/io/gitlab/arturbosch/detekt/idea/config/DetektPluginSettings.kt

```diff
@@ -1,13 +1,8 @@
 package io.gitlab.arturbosch.detekt.idea.config
 
-import com.intellij.openapi.components.BaseState
-import com.intellij.openapi.components.Service
-import com.intellij.openapi.components.SimplePersistentStateComponent
-import com.intellij.openapi.components.State
-import com.intellij.openapi.components.Storage
-import com.intellij.openapi.components.service
+import com.intellij.openapi.components.*
```

Comment:
Hm, it seems I have not checked in the code style to remove wildcard imports.

Reply:
I was surprised, too :) but decided to use whatever code style was default in the project